index.html

```diff
@@ -1,19 +1,723 @@
-<!DOCTYPE HTML>
+﻿<!DOCTYPE HTML>
 <html lang="pl">
+
 <head>
 	<meta charset="utf-8" />
 	<title>Monitory komputerowe</title>
 	<meta name="description" content="Zobacz wszystko, co powienieneś wiedzieć o monitorach komputerowych w technikum informatycznym." />
 	<meta name="keywords" content="budowa, zasada działania, CRT, LCD, złącza, wady, zalety, parametry monitorów, oznaczenia, maski, rodzaje matryc: TN, PVA, MVA, IPS, wymiar przekątnej, rozdzielczość, wielkość plamki, wielkość piksela, jasność, kontrast, bad pixel, PDP, OLED, VGA, DSUB, HDMI, DVI, jaki monitor kupić, porównanie CRT vs LCD" />
 	<meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1" />
-	
 	<link rel="stylesheet" href="style.css" type="text/css" />
-	
+	<link href="https://fonts.googleapis.com/css?family=Open+Sans:400,700&amp;subset=latin-ext" rel="stylesheet">
+	<script src="https://ajax.googleapis.com/ajax/libs/jquery/2.2.4/jquery.min.js"></script>
+	<script src="scripts/script.js"></script>
+	<script src="scripts/jquery.scrollTo.min.js"></script>
 </head>
 
 <body>
-	
-	
-	
+	<div id="logo">
```

Comment:
This file must be rearranged:
1) Reformat it: use 4 spaces indention
2) Dont use inline styles, define css class instead and use it in this template
3) don't use polish language in places where you don't have to, like images file names
4) Instead of using <br/>, define <p> paragraph tag and style it properly

Reply:
1 - done. 3-4 refers to index.html lines 66 to 717. Code between those lines is not to be reviewed. It's not the goal of this project as mentioned in readme file. Regular project would have styles placed in css.